wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -247,6 +273,127 @@ sendMoney pm txpConfig submitTx passphrase moneySource dstDistr policy = do
     logDebug "sendMoney: constructing response"
     fst <$> constructCTx ws' srcWallet srcWalletAddrsDetector diff th
 
+-- | Create new raw transaction which will be signed externally (for example, by Ledger device).
+createNewUnsignedTransaction
+    :: (MonadWalletTxFull ctx m)
+    => ProtocolMagic
+    -> MoneySource
+    -> NonEmpty (CId Addr, Coin)
+    -> InputSelectionPolicy
+    -> Address
+    -> m (Tx, NonEmpty (Address, [Word32]))
+createNewUnsignedTransaction pm moneySource dstDistr policy changeAddress = do
+    when walletTxCreationDisabled $
+        throwM err405
+        { errReasonPhrase = "Transaction creation (including unsigned ones) is disabled by configuration!"
+        }
+
+    db <- askWalletDB
+    ws <- getWalletSnapshot db
+
+    addrMetas' <- getMoneySourceAddresses ws moneySource
+    addrMetas <- nonEmpty addrMetas' `whenNothing`
+        throwM (RequestError "Unsigned transaction: Given money source has no addresses!")
+
+    let srcAddrs = map (view wamAddress) addrMetas
+        srcAddrsDerivationPaths = map (\meta -> [_wamAccountIndex meta, _wamAddressIndex meta]) addrMetas
+        srcAddrsInfo = NE.zip srcAddrs srcAddrsDerivationPaths
+
+    logDebug "createNewUnsignedTransaction: processed addrs"
+
+    _ <- getSomeMoneySourceAccount ws moneySource
+    outputs <- coinDistrToOutputs dstDistr
+    let pendingAddrs = getPendingAddresses ws policy
+    rewrapTxError "Cannot create unsigned transaction" $
+        prepareUnsignedTx pm pendingAddrs policy srcAddrs outputs changeAddress >>= \case
+            Left txError ->
+                throwM (RequestError $ show txError)
```

Comment:
I think this is going to bypass the `rewrapTxError` logic. Is that the desired behavior?